tycho-its/projects/issue23/bundle/build.properties

```diff
@@ -0,0 +1,5 @@
+source.. = src/
+output.. = bin/
+bin.includes = .,\
+               META-INF/
+additional.bundles = org.eclipse.emf.codegen.ecore
```

Comment:
unfortunately the problem is not reproducible with a dependency closer to platform

Reply:
I found org.junit useful for this purpose as it is part of the sdk but I think it does not mater much as long as we do not need to include large dependency chains.